ci/lava/tests/rootfs-update.sh

```diff
@@ -127,28 +118,30 @@ else
                 fi
 
             else
-
-
                 # Now copy the tar file to the DUT
                 $mbl_command put /tmp/payload.tar /scratch
 
                 # Now update the rootfs - the -s prevents the automatic reboot
                 $mbl_command shell 'su -l -c "mbl-firmware-update-manager /scratch/payload.tar -v --keep --assume-no"'
 
                 # Now reboot the board and get the result of the reboot command
-                $mbl_command shell 'su -l -c "reboot || echo $?"'
+                # $mbl_command shell 'su -l -c "reboot || echo $?"'
+                # mbl-cli doesn't close the connection properly when we run a reboot: IOTMBL-2389
+                # Until we fix it, we reboot the board sending the command over ssh with the appropriate
+                # options ServerAliveInterval=10 -o ServerAliveCountMax=3
+                ssh_options=("-o" "StrictHostKeyChecking=no" "-o" "ServerAliveInterval=5" "-o" "ServerAliveCountMax=1")
```

Comment:
StrictHostKeyChecking means "man-in-the-middle attack" warnings are printed in the log. Consider removing is.

Reply:
Cannot remove it otherwise the ssh client waits for a user confirmation. This has been done on purpose. If you want I can add a comment to it.